scipy/stats/morestats.py

```diff
@@ -1084,7 +1092,29 @@ def boxcox_normmax(x, brack=(-2.0, 2.0), method='pearsonr'):
         'all'
             Use all optimization methods available, and return all results.
             Useful to compare different methods.
-
+    bounds : sequence, optional
+        For `optimize_method` 'bounded', `bounds` is mandatory and must have two items
+        corresponding to the optimization bounds.
+    optimize_method : str or callable, optional
+        Type of solver.  Should be one of:
+
+        - 'Brent'     :ref:`(see here) <optimize.minimize_scalar-brent>`
+        - 'Bounded'   :ref:`(see here) <optimize.minimize_scalar-bounded>`
+        - 'Golden'    :ref:`(see here) <optimize.minimize_scalar-golden>`
+        - custom - a callable object
+
+    tol : float, optional
+        Tolerance for termination. For detailed control, use solver-specific
+        options. See `scipy.optimize.minimize_scalar` for more details.
+    options : dict, optional
+        A dictionary of solver options.
+
+        maxiter : int
+            Maximum number of iterations to perform.
+        disp : bool
+            Set to True to print convergence messages.
+
+        See `scipy.optimize.minimize_scalar` for more details.
```

Comment:
```suggestion
        See `scipy.optimize.minimize_scalar` for more details.

```
"Returns" is not rendering correctly without this space.